app/src/main/java/com/dessalines/thumbkey/keyboards/ITTypeSplit.kt

```diff
@@ -547,7 +574,7 @@ val KB_IT_TYPESPLIT_SHIFTED = KeyboardC(
 )
 
 val KB_IT_TYPESPLIT: KeyboardDefinition = KeyboardDefinition(
-    title = "italiano type-split",
+    title = "IT Type-Split italiano",
```

Comment:
Change back to the old title.

Reply:
sorry @dessalines, I have updated the line changing it back but I don't know what I should do now

Reply:
Its no problem, I'll clean this and the other PRs up when I do the next release, just keep it open for now.